Makefile

```diff
@@ -215,12 +215,12 @@ gitserver:
 # ref: https://stackoverflow.com/questions/33509194/command-to-delete-all-pods-in-all-kubernetes-namespaces
 .PHONY: e2e-test-clean
 e2e-test-clean:
-	$(KUBECTL) delete -n numaplane-e2e isbsvc --all
-	$(KUBECTL) delete -n numaplane-e2e pipelines --all
-	$(KUBECTL) delete -n numaplane-e2e cm --all
-	$(KUBECTL) delete -n numaplane-e2e secret --all
-	$(KUBECTL) delete -n numaplane-e2e all --all
-	$(KUBECTL) delete -n numaplane-system pod --all
+	$(KUBECTL) delete -n numaplane-e2e isbsvc --all || true
```

Comment:
Why do we need this?

Reply:
when e2e tests are run and there are no pods running for isbvc or secret is created it exits the test ,so to prevent exiting i have added this

Reply:
I'm not sure I understand. Is this when calling `make e2e-test-start`? (which calls this as a dependency) It seems that if you try to delete something that isn't there, then it would be okay and just keep going, no?

Reply:
Removed this as discussed on slack chat